cvat/apps/engine/views.py

```diff
@@ -189,6 +189,9 @@
 _DATA_UPDATED_DATE_HEADER_NAME = 'X-Updated-Date'
 _RETRY_AFTER_TIMEOUT = 10
 
+def get_logo_uri(request) -> str:
```

Comment:
This seems like it should be a private method inside `ServerViewSet` (or just inlined).

Reply:
Feels like inlined would be simplier. Added.